eduiddashboard/vccs.py

```diff
@@ -41,12 +41,12 @@ def check_password(vccs_url, password, user, vccs=None):
     if isinstance(user, DashboardLegacyUser):
         user = DashboardUser(data=user._mongo_doc)
 
-    for cred in user.passwords.to_list():
+    for cred in user.credentials.to_list():
```

Comment:
We need to use user.credentials.filter(Password).to_list() to just get a list of the user passwords.